src/Microsoft.Health.Fhir.TemplateManagement/Client/IOrasClient.cs

```diff
@@ -3,14 +3,15 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Microsoft.Health.Fhir.TemplateManagement.Client
 {
     public interface IOrasClient
     {
-        Task PullImageAsync(string outputFolder);
+        Task<string> PullImageAsync(string outputFolder);
 
-        Task PushImageAsync(string inputFolder);
+        Task PushImageAsync(string inputFolder, List<string> filePathList);
```

Comment:
Can we add a new interface instead of replacing the older one? I'm afraid this method could change in the future.

Reply:
Actually, we hope user don't use the old interface, since the old interface can not guarantee the order of layers which will lead to wrong manifest.